eng/Versions.props

```diff
@@ -10,7 +10,7 @@
     <MicrosoftDotNetBuildTasksTemplatingPackageVersion>10.0.0-beta.24429.6</MicrosoftDotNetBuildTasksTemplatingPackageVersion>
     <MicrosoftTestingExtensionsCodeCoverageVersion>17.13.1</MicrosoftTestingExtensionsCodeCoverageVersion>
     <!-- comment to facilitate merge conflicts -->
-    <MicrosoftTestingInternalFrameworkVersion>1.4.3</MicrosoftTestingInternalFrameworkVersion>
-    <MSTestEngineVersion>1.0.0-alpha.24562.1</MSTestEngineVersion>
+    <MicrosoftTestingInternalFrameworkVersion>1.4.3-preview.24610.1</MicrosoftTestingInternalFrameworkVersion>
+    <MSTestEngineVersion>1.0.0-alpha.24610.1</MSTestEngineVersion>
```

Comment:
If there is a possibility that we may need to ship a future service release for 3.6, this change should be reverted as this version isn't on NuGet.org.